config/initializers/types.rb

```diff
@@ -1 +1,3 @@
-ActiveRecord::Type.register(:app_version, AppVersionType)
+Rails.application.config.to_prepare do
```

Comment:
This change comes from https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#autoloading-during-initialization. A couple of alternatives are listed in https://guides.rubyonrails.org/v7.0/autoloading_and_reloading_constants.html#autoloading-when-the-application-boots, but I'm not sure if any of the other use cases are a better fit.